services/tenant-ui/frontend/src/store/innkeeper/innkeeperOidcStore.ts

```diff
@@ -0,0 +1,111 @@
+import axios from 'axios';
+import { defineStore, storeToRefs } from 'pinia';
+import { computed, ref } from 'vue';
+import { useConfigStore } from '../configStore';
+import { useInnkeeperTokenStore } from './innkeeperTokenStore';
+import { UserManager } from 'oidc-client-ts';
+
+export const useInnkeeperOidcStore = defineStore('innkeeperOidcStore', () => {
+  // other stores
+  const { config } = storeToRefs(useConfigStore());
+  const { token } = storeToRefs(useInnkeeperTokenStore());
+
+  // private (move to other file maybe?)
+  const _settings: any = {
```

Comment:
Something tells me the linter will complain about variables/functions starting with an underscore.

Reply:
Lint and code climate seem to be ok with it, pretty standard for "private" vars in JS I think.